contracts/reflect/src/contract.rs

```diff
@@ -460,7 +462,10 @@ mod tests {
         let qres: Reply = from_json(raw).unwrap();
         assert_eq!(qres.id, id);
         let result = qres.result.unwrap();
-        assert_eq!(result.data, Some(data));
+        #[allow(deprecated)]
+        {
+            assert_eq!(result.data, Some(data));
+        }
```

Comment:
Isn't this working as well?
```suggestion
        #[allow(deprecated)]
        assert_eq!(result.data, Some(data));
```

Reply:
No, that does not work. For some reason attributes are ignored when applied to macro invocations.